app/assets/javascripts/components/common/user_settings.es6.jsx

```diff
@@ -33,6 +34,22 @@ class UserSettings extends React.Component {
     );
   }
 
+  attemptTeacherSave(resolve, reject) {
+    let status;
```

Comment:
It's called in the teacher_profile when you attemptSave in the EditableInputGroup. It's just because we needed a separate route for the teacher update. Is that ok?